MakingItBetterG55/src/application/OptionsMenu.java

```diff
@@ -0,0 +1,179 @@
+package application;
+
+import javafx.application.Application;
+import javafx.stage.Stage;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.ColumnConstraints;
+import javafx.scene.layout.GridPane;
+import javafx.scene.layout.RowConstraints;
+import javafx.scene.paint.Color;
+import javafx.scene.text.Font;
+import javafx.scene.text.FontWeight;
+import javafx.scene.text.Text;
+import javafx.scene.input.KeyCode;
+import javafx.scene.input.KeyEvent;
+import javafx.scene.layout.Pane;
+import javafx.scene.shape.Rectangle;
+
+public abstract class OptionsMenu extends Application{

```

Comment:
All looks really good Emily! Could you switch from extending Application and having a separate stage for the options screen to creating and returning a scene and then in Main/ anywhere the options button is clicked just set scene to your scene? This is how we're doing the rest of the screen changes and I just want to be consistent